runtime/intern.c

```diff
@@ -792,32 +789,45 @@ static void caml_parse_header(struct caml_intern_state* s,
    when the memory block for the compressed input can be freed
    before more memory is allocated. */
 
+size_t (*caml_intern_decompress_input)(unsigned char *,
+                                       uintnat,
+                                       const unsigned char *,
+                                       uintnat) = NULL;
+
+CAMLprim value caml_compression_available(value vunit)
```

Comment:
This is not used, is it intended ?

Reply:
Yes, indeed. To avoid an unnecessary bootstrap, the primitive will be removed before at release.